config/settings/production.yml

```diff
@@ -8,4 +8,4 @@ manage_backend:
 manage_ui:
   base_url: https://www.publish-teacher-training-courses.service.gov.uk
```

Comment:
@fofr is this one still correct then?

Reply:
I think this one's fine – basically we are using the service domain for Publish.